### PR TITLE
chore(RHTAPWATCH-1297): Add rpms-signature-scan to template

### PR DESCRIPTION
**Pull request by avi-biton**
Add rpms-signature-scan

# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

**Comment by avi-biton** [comment]
Please merge https://github.com/konflux-ci/build-definitions/pull/1482 before merging this PR.
I added a fix to the readme file and I want to rerun the `/hack/generate-pipelines-readme.py ` script to pick up the changes

**Comment by avi-biton** [comment]
https://github.com/konflux-ci/build-definitions/pull/1482 was merged.
I rebased and rerun the `/hack/generate-pipelines-readme.py` script

File: pipelines/template-build/template-build.yaml [hunk 1]
```diff
@@ -265,7 +265,21 @@ spec:
       workspaces:
       - name: workspace
         workspace: workspace
-
+    - name: rpms-signature-scan
```
Comment (mmorhun) [inline pipelines/template-build/template-build.yaml:268]: Do we really need this task for all pipelines and all users?

Comment (lcarva) [reply pipelines/template-build/template-build.yaml:268]: I believe so. It's how Konflux will be able to enforce that all included RPMs are signed with the right key via EC.

Comment (avi-biton) [reply pipelines/template-build/template-build.yaml:268]: Yes, @mmorhun thats the feature.
This is the [refinement doc](https://docs.google.com/document/d/1fhwCyXKU8kB5cX4UkxAJ51lEbF_vXG7APTICusU8AsU/edit)
This is the [feature](https://issues.redhat.com/browse/KONFLUX-3149)

cc @ralphbean @lcarva

**Pull request merged**